chain-spec-generator/src/system_parachains_specs.rs

```diff
@@ -0,0 +1,552 @@
+) -> asset_hub_kusama_runtime::RuntimeGenesisConfig {
+	asset_hub_kusama_genesis(
+		// initial collators.
+		wasm_binary,
+		invulnerables(),
+		testnet_accounts(),
+		1000.into(),
+	)
+}
+
+pub fn asset_hub_kusama_local_testnet_config() -> Result<Box<dyn ChainSpec>, String> {
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("ss58Format".into(), 2.into());
+	properties.insert("tokenSymbol".into(), "KSM".into());
+	properties.insert("tokenDecimals".into(), 12.into());
+
+	let wasm_binary =
+		asset_hub_kusama_runtime::WASM_BINARY.ok_or("AssetHubKusama wasm not available")?;
+
+	Ok(Box::new(AssetHubKusamaChainSpec::from_genesis(
+		// Name
+		"Kusama Asset Hub Local",
+		// ID
+		"asset-hub-kusama-local",
+		ChainType::Local,
+		move || asset_hub_kusama_local_genesis(wasm_binary),
+		Vec::new(),
+		None,
+		None,
+		None,
+		Some(properties),
+		Extensions { relay_chain: "kusama-local".into(), para_id: 1000 },
+	)))
+}
+
+// CollectivesPolkadot
+fn collectives_polkadot_genesis(
+	wasm_binary: &[u8],
+	invulnerables: Vec<(AccountId, AuraId)>,
+	endowed_accounts: Vec<AccountId>,
+	id: ParaId,
+) -> collectives_polkadot_runtime::RuntimeGenesisConfig {
+	collectives_polkadot_runtime::RuntimeGenesisConfig {
+		system: collectives_polkadot_runtime::SystemConfig {
+			code: wasm_binary.to_vec(),
+			..Default::default()
+		},
+		balances: collectives_polkadot_runtime::BalancesConfig {
+			balances: endowed_accounts
+				.iter()
+				.cloned()
+				.map(|k| (k, COLLECTIVES_POLKADOT_ED * 4096))
+				.collect(),
+		},
+		parachain_info: collectives_polkadot_runtime::ParachainInfoConfig {
+			parachain_id: id,
+			..Default::default()
+		},
+		collator_selection: collectives_polkadot_runtime::CollatorSelectionConfig {
+			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
+			candidacy_bond: COLLECTIVES_POLKADOT_ED * 16,
+			..Default::default()
+		},
+		session: collectives_polkadot_runtime::SessionConfig {
+			keys: invulnerables
+				.into_iter()
+				.map(|(acc, aura)| {
+					(
+						acc.clone(),                             // account id
+						acc,                                     // validator id
+						collectives_polkadot_session_keys(aura), // session keys
+					)
+				})
+				.collect(),
+		},
+		// no need to pass anything to aura, in fact it will panic if we do. Session will take care
+		// of this.
+		aura: Default::default(),
+		aura_ext: Default::default(),
+		parachain_system: Default::default(),
+		polkadot_xcm: collectives_polkadot_runtime::PolkadotXcmConfig {
+			safe_xcm_version: Some(SAFE_XCM_VERSION),
+			..Default::default()
+		},
+		alliance: Default::default(),
+		alliance_motion: Default::default(),
+	}
+}
+
+fn collectives_polkadot_local_genesis(
+	wasm_binary: &[u8],
+) -> collectives_polkadot_runtime::RuntimeGenesisConfig {
+	collectives_polkadot_genesis(
+		// initial collators.
+		wasm_binary,
+		invulnerables(),
+		testnet_accounts(),
+		1002.into(),
```

Comment:
@NachoPal 
I know it is just for local testing, but why 1002 for collectives-polkadot and 1003 for BridgeHubs?
we use hard-coded constants for system para ids e.g. [here](https://github.com/polkadot-fellows/runtimes/blob/main/relay/polkadot/constants/src/lib.rs#L118-L119)  (BHs are coming in separate PRs)
so this will complicate local runs/testing.

I would suggest to align paraIds here with real deployment:
![image](https://github.com/polkadot-fellows/runtimes/assets/8159517/6bd0c78b-75a4-44de-8eb4-231c90578ef4)

Reply:
Fixed